app/controllers/api/move_events_controller.rb

```diff
@@ -8,7 +8,7 @@ class MoveEventsController < ApiController
     before_action :validate_idempotency_key
     around_action :idempotent_action
 
-    APPROVE_PARAMS = [:type, attributes: %i[timestamp date]].freeze
+    APPROVE_PARAMS = [:type, attributes: %i[timestamp date create_in_nomis]].freeze
```

Comment:
This feels like a weird place for a special snowflake optional parameter. How do you feel about a query param or making it part of meta? The resource attributes usually map directly onto the entity attributes in the database, etc?

Reply:
Not sure I agree. This will be a checkbox on the form that the user ticks if they want the event created in Nomis, or leave unticked if they don't. So it seems reasonable to treat this in the same way as the date input on the same form.

Reply:
Beyond a subjective sense of how things should be I don't really have much more to say 🎉 .